src/readme.ts

```diff
@@ -17,9 +17,15 @@ export class ReadmeFile extends FileBase {
   }
 
   protected synthesizeContent(resolver: IResolver) {
-    const { providerName, providerVersion, packageInfo } = resolver.resolve(
-      this.options
-    ) as ReadmeFileOptions;
+    const { providerName, fqproviderName, providerVersion, packageInfo } =
+      resolver.resolve(this.options) as ReadmeFileOptions;
+
+    const fqpnURL = fqproviderName.includes("/")
+      ? fqproviderName
+      : `hashicorp/${fqproviderName}`;
+    const versionURL = providerVersion
+      ? providerVersion.replace("~> ", "").concat(".0")
```

Comment:
I'm unsure if we always require a trailing space after `~>`. Maybe replacing with a regex is better here? `/~>\s*/`?

Reply:
Good catch, regex here sounds far more robust.